index.bs

```diff
@@ -206,268 +177,455 @@ interface ImageCapture {
 
 </dl>
 
-
-# PhotoCapabilities
+# PhotoCapabilities # {##photocapabilities-section}
 
 <pre class="idl">
   interface PhotoCapabilities {
-    readonly attribute MeteringMode       whiteBalanceMode;
-    readonly attribute MediaSettingsRange colorTemperature;
-    readonly attribute MeteringMode       exposureMode;
-    readonly attribute MediaSettingsRange exposureCompensation;
-    readonly attribute MediaSettingsRange iso;
-    readonly attribute boolean            redEyeReduction;
-    readonly attribute MeteringMode       focusMode;
-
-    readonly attribute MediaSettingsRange brightness;
-    readonly attribute MediaSettingsRange contrast;
-    readonly attribute MediaSettingsRange saturation;
-    readonly attribute MediaSettingsRange sharpness;
+    readonly attribute FillLightMode      fillLightMode;
     readonly attribute MediaSettingsRange imageHeight;
     readonly attribute MediaSettingsRange imageWidth;
-    readonly attribute MediaSettingsRange zoom;
-    readonly attribute FillLightMode      fillLightMode;
+    readonly attribute boolean            redEyeReduction;
   };
 </pre>
 
 ## Attributes ## {##photocapabilities-attributes}
 
 <dl class="domintro">
-  <dt><dfn attribute for="PhotoCapabilities"><code>whiteBalanceMode</code></dfn></dt>
-  <dd>This reflects the current <a>white balance mode</a> setting. </dd>
+  <dt><dfn attribute for="PhotoCapabilities"><code>redEyeReduction</code></dfn></dt>
+  <dd>This reflects whether <a>red eye reduction</a> is suported, and is boolean - true means supported.</dd>
```

Comment:
It's unlikely there isn't some switch to disable the red-eye reduction because it implies a pre-flash that takes a significant amount of time compared to the exposure time; however it might be a good insurance policy for the future. In that case it could read, inspired by `echoCancellation`:

```
If the source cannot do <a>red eye reduction</a> a single false is reported. If 
<a>red eye reduction</a> cannot be turned off, a single true is reported. If the 
script can control the feature, the source reports a list with both true and false 
as possible values.
```
with the interface entry being
`readonly attribute sequence<boolean> redEyeReduction;`

other opinions, @gmandyam ?